docs/tools/flow-cli/flix.md

```diff
@@ -1,35 +1,124 @@
 ```
 
 ```shell
 # Execute a FLIX script by id on Testnet
-flow flix bd10ab0bf472e6b58ecc0398e9b3d1bd58a4205f14a7099c52c0640d9589295f --network testnet
+flow flix execute bd10ab0bf472e6b58ecc0398e9b3d1bd58a4205f14a7099c52c0640d9589295f --network testnet
 ```
 
 ```shell
 # Execute a local FLIX script by path on Testnet
-flow flix ./multiply.template.json 2 3 --network testnet
+flow flix execute ./multiply.template.json 2 3 --network testnet
+```
+
+The Flow CLI provides a `flix` command to `package` up generated plain and simple JavaScript. This JavaScript uses FCL (Flow Client Library) to call the cadence the Flow Interaction Templates (FLIX) is based on. 
+
+<Callout type="info">
+Currently, flix `package` command only supports generating FCL (Flow Client Library) specific JavaScript, there are plans to support other languages like golang.
+</Callout>
+
+
+```shell
+flow flix package <query> [flags]
+```
+
+## Package
+
+Queries can be a FLIX `id`, `name`, or `path` to a local FLIX file. This command leverages [@onflow/fcl](https://www.npmjs.com/package/@onflow/fcl) to ingest a template that has Cadence code. 
+
+### Package Usage
+
+```shell
+# Generate packaged code that leverages FCL to call the Cadence transaction code, `--save` flag will save the output to a specific file
+flow flix package transfer-flow --save ./package/transfer-flow.js
 ```
 
+```shell
+# Geneate package code for a FLIX script using id, since there is no saving file, the result will display in terminal
+flow flix package bd10ab0bf472e6b58ecc0398e9b3d1bd58a4205f14a7099c52c0640d9589295f 
+```
+
+```shell
+# Generate package code using local template file to save in a local file 
+flow flix package ./multiply.template.json --save ./multiply.js
+```
+
+### Example package output
+```shell
+flow flix package transfer-flow
+```
+
+```javascript
+
+/**
+    This binding file was auto generated based on FLIX template v1.0.0. 
+    Changes to this file might get overwritten.
+    Note fcl version 1.3.0 or higher is required to use templates. 
+**/
+
+import * as fcl from "@onflow/fcl"
+const flixTemplate = "transfer-flow"
+
+/**
+* Transfer tokens from one account to another
+* @param {Object} Parameters - parameters for the cadence
+* @param {string} Parameters.amount - The amount of FLOW tokens to send: UFix64
+* @param {string} Parameters.to - The Flow account the tokens will go to: Address
+* @returns {Promise<string>} - returns a promise which resolves to the transaction id
+*/
+export async function transferTokens({amount, to}) {
+  const transactionId = await fcl.mutate({
+    template: flixTemplate,
+    args: (arg, t) => [arg(amount, t.UFix64), arg(to, t.Address)]
+  });
+
+  return transactionId
+}
+```
+
+<Callout type="info">
+SOON: `flix generate` will generate FLIX json files using FLIX specific properties in comment blocks directly in Cadence code, [FLIP - Interaction Template Cadence Doc](https://github.com/onflow/flips/pull/80)
```

Comment:
![Screenshot 2023-11-01 at 10 37 35 AM](https://github.com/onflow/docs/assets/1666730/de80bcb0-963d-4b9d-9e49-05e52a23055a)

Backticks don't seem to work in a callout

Reply:
must be a space or something, played around with it and it seems to be working.

![2023-11-01 15 12 56](https://github.com/onflow/docs/assets/3970376/9e2c1966-30ad-4a52-b031-80a3176c03e0)

Reply:
Nice much better